braidpool_spec.md

```diff
@@ -146,14 +229,38 @@ primarily to discourage pool hopping. We don't feel that this is needed in the
 modern day and a smoothing function applied to payouts interferes with the
 notion of using shares as a hashrate derivative instrument.
 
-A purely work-weighted proportional algorithm would work for a pure-DAG
+[kp]
+
+I like PPLNS as it does exactly that - prevents pool hopping. By not addressing
+this we are leaving braidpool vulnerable to future innovations in markets and
+pool operations. Why do you feel this is not a problem anymore. If we want to
+leave PPLNS out of braidpool, we need a more convincing argument. I'd argue for
+PPLNS and reference Meni's work.
+
+You could say, braidpool allows for any accounting scheme on top of the
+eventually consistent view of shares. On that note, we should talk about the
+eventually consistency of shares view in the broadcast section. This again makes
+me think we need that overview section with a system architecture diagram where
+each component has clearly defined responsibilities.
+
+[/kp]
```

Comment:
I claim that none of the pool hopping algorithms actually work. As such I'm trying to keep it simple because there's a very wide space of reward redistribution that can become very political without actually solving the claimed problem (pool hopping). Also as the pool doesn't store funds, most payout mechanisms are not available to us (e.g. PPS and variants which are most popular now).

So IMHO it's best to use the simplest solution, and NOT address the pool hopping problem. If you think you can solve it the go ahead and try, but this has consumed the minds of many without any satisfactory solution.